RELEASENOTES.md

```diff
@@ -1,3 +1,16 @@
-### 4.0.1 (Dec 27, 2017)
-* [CB-13705](https://issues.apache.org/jira/browse/CB-13705) Fix to allow 4.0.0 version install
+# Changelog
```

Comment:
Release notes are not the same as a changelog file. Please rename this file to CHANGELOG.md and leave the upstream file untouched.

Reply:
Fixed!